x-pack/solutions/security/plugins/security_solution/server/lib/telemetry/tasks/configuration.ts

```diff
@@ -101,29 +101,31 @@ export function createTelemetryConfigurationTaskConfig() {
         }
 
         if (configArtifact.pagination_config) {
-          log.l('Updating pagination configuration');
+          log.info('Updating pagination configuration');
           telemetryConfiguration.pagination_config = configArtifact.pagination_config;
           _receiver.setMaxPageSizeBytes(configArtifact.pagination_config.max_page_size_bytes);
           _receiver.setNumDocsToSample(configArtifact.pagination_config.num_docs_to_sample);
         }
 
         if (configArtifact.indices_metadata_config) {
-          log.l('Updating indices metadata configuration');
+          log.info('Updating indices metadata configuration');
           telemetryConfiguration.indices_metadata_config = configArtifact.indices_metadata_config;
         }
 
         if (configArtifact.ingest_pipelines_stats_config) {
-          log.l('Updating ingest pipelines stats configuration');
+          log.info('Updating ingest pipelines stats configuration');
           telemetryConfiguration.ingest_pipelines_stats_config =
             configArtifact.ingest_pipelines_stats_config;
         }
 
         await taskMetricsService.end(trace);
 
-        log.l('Updated TelemetryConfiguration', { configuration: telemetryConfiguration });
+        log.info('Updated TelemetryConfiguration', {
+          configuration: telemetryConfiguration,
+        } as LogMeta);
```

Comment:
The `configuration` object is quite large. Can we lower the level to `debug` (or even `trace`)?

If these logs are indexed, they add a lot of fields to the mappings and to the dataview ([internal example](https://overview.elastic-cloud.com/app/r/s/XpFr5)). 
Q: Do we need to index them?

Reply:
you are right, we don't need to do it, just logging in `debug` the event is enough.